kiwmi/luak/kiwmi_server.c

```diff
@@ -241,6 +241,36 @@ l_kiwmi_server_unfocus(lua_State *L)
     return 0;
 }
 
+static int
+l_kiwmi_server_verbose(lua_State *L)
+{
+    luaL_checkudata(L, 1, "kiwmi_server");
+    luaL_checktype(L, 2, LUA_TNUMBER);
+
+    int verbosity = lua_tointeger(L, 2);
+
+    if (verbosity >= WLR_SILENT && verbosity < WLR_LOG_IMPORTANCE_LAST) {
```

Comment:
Maybe just clamp the values here? That's closer to what's happening with -VVVVVVV spam as well.

Reply:
Seems to be a better solution. Done.